section-09-differential-testing/model-package/differential_tests/test_model_accuracy.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import numpy as np
+from regression_model.predict import make_prediction
+
+
+def test_make_prediction(sample_input_data, persist_results):
```

Comment:
Not sure what is going on here.

The script is called test_model_accuracy, but we are not doing that. In fact, it would be a great test, that the accuracy of the recreated model falls within a tolerance from the one observed in the research environment, which is found in cell 10 of [this notebook](https://github.com/trainindata/deploying-machine-learning-models/blob/master/section-04-research-and-development/08-final-machine-learning-pipeline.ipynb):



```
train mse: 781396630
train rmse: 27953
train r2: 0.8748530315439078

test mse: 1060769014
test rmse: 32569
test r2: 0.8456415571208442
```

Maybe we could add that test? 

Also let's not call it model "accuracy", because that is for classification, let's call it model "performance".

Reply:
agreed, renamed to `test_generate_predictions`